plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerArtifact.kt

```diff
@@ -57,8 +59,9 @@ open class CodeModernizerArtifact(
                 }
                 val patches = extractPatches(manifest)
                 val summary = extractSummary(manifest)
+                val summaryMarkdownFile = getSummaryFile(manifest)
                 if (patches.size != 1) throw RuntimeException("Expected 1 patch, but found ${patches.size}")
-                return CodeModernizerArtifact(zipPath, manifest, patches, summary)
+                return CodeModernizerArtifact(zipPath, manifest, patches, summary, summaryMarkdownFile)
```

Comment:
Used so that we can open the `summary.md` file directly, which will natively support file path clicks and other formatting work Lin is doing.